kairos-contracts/demo-contract/deposit-session/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "deposit-session"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+casper-contract = {git="https://github.com/cspr-rad/casper-node", branch="kairos-demo-1.5.6", features=["test-support"]}
```

Comment:
Why are we picking this specific branch of `1.5.6`? why not upstream `1.5.9`

Reply:
Because I added the risc0 host function verifier to this branch.

Reply:
If we want to be able to verify the proof on the L1 then we have to use this branch that I created.

Reply:
And why is it 1.5.6?

Reply:
It was 1.5.6 before. There is no 1.5.9, 1.5.6 is the latest stable release of 1.5.x.

Reply:
Resolve this, 1.5.6 is the latest stable release and what we all agreed to do the demo on

Reply:
Oh sorry about that, my fault